basiclti/basiclti-common/src/java/org/sakaiproject/basiclti/util/SakaiBLTIUtil.java

```diff
@@ -2857,4 +2842,47 @@ public static void addEnabledCapability(JSONArray enabledCapabilities, String ca
 		}
 	}
 
+	/**
+	 *  Check if we are an LTI 1.3 launch or not
+	 */
+	public static boolean isLTI13(Map<String, Object> tool, Map<String, Object> content) {
+		// 0=inherit from tool, 1=LTI 1.1, 2=LTI 1.3
+		if ( content != null ) {
+			Long contentLTI13 = getLong(content.get(LTIService.LTI13));
```

Comment:
can this be null?

Reply:
No - getLong returns -1 instead of null.